lib/internal/blocklist.js

```diff
@@ -65,6 +74,15 @@ class BlockList {
       rules: this.rules,
     }, opts)}`;
   }
+  loadFromFile=(savePath="./blocklist.json")=>{
```

Comment:
```suggestion

  loadFromFile(savePath="./blocklist.json") {
```

Why make this an arrow function?

Instead of building the file loading directly into `BlockList`, consider having `BlockList` instead generate a `Buffer` or be created from a `Buffer` and let the application do the file handling itself.

```
const blockList = createBlockListSomehow();
const buf = blockList.save();
fs.writeFileSync('blocklist.json', buf);
```

Reply:
@jasnell  done